pages/common/youtube-dl.md

```diff
@@ -19,6 +19,10 @@
 
 `youtube-dl -x --audio-format {{mp3}} {{url}}`
 
+- Download the best quality audio and video and merge them:
+
+`youtube-dl -f {{bestvideo+bestaudio}} {{url}}`
```

Comment:
I'm really not sure how is this supposed to work. Have you meant to put something else in place of `{{bestvideo+bestaudio}}`?

Reply:
This works because `bestvideo` and `bestaudio` are the actual values. 👍 ([Docs](https://github.com/ytdl-org/youtube-dl#format-selection))

Reply:
Shouldn't the braces be removed then?

Reply:
I am bit confused. I think the braces should be removed, as @zdroid commented.

Reply:
I removed the braces.

Reply:
I'm not sure that they should, @zdroid, @ivanhercaz. `bestvideo+bestaudio` can be replaced with other values.

Reply:
They can, but the example was directed at this. If there are other available values, then `{{value1|value2|value3}}` syntax should have been used instead.